docs/document/docs/releases/release-notes-v1.0.1.md

```diff
@@ -0,0 +1,18 @@
+---
+id: release-notes-v1.0.1
+title: v1.0.1 Release Notes
+sidebar_label: v1.0.1 Release Notes
+original_id: release-notes-v1.0.1
```

Comment:
I think it is no need to specify `original_id` under docs folder.
This is for versioning.

Reply:
Ok, i removed it